README.MD

```diff
@@ -12,6 +12,8 @@ GitHub allows you to [co-author commits](https://help.github.com/articles/creati
 
 To speed up your workflow you can preinstall co-commit using `npm i -g co-commit`.
 
+You can also utilize git flags with co-commit: eg `git commit --force` becomes `npx commit --force`. Additionally, you can skip prompts using flags: eg: `-m "your commit message"` or `-c "mariiapunda, juliettepretot"`.
```

Comment:
`--force` isn't actually a valid flag when committing, will update this to `--all`